zoe-cli/src/config/config.kt

```diff
@@ -160,7 +160,7 @@ enum class Format {
 
     fun format(content: JsonNode, foreach: (String) -> Unit) = when (this) {
         Raw -> foreach(content.toString())
-        Json -> foreach(content.toString())
+        Json -> foreach(content.toPrettyString())
```

Comment:
Can you create a new enum like `JsonPretty`? because the non pretty printed for is still useful to have a compact representation of the json

Reply:
ok yes that's fine. What would you like as identifier for the pretty json version?
json-pretty or short: json-p?

Reply:
`json-p` looks good :+1:

Reply:
Ok! I was thinking about another PR where one can configure a default output-format. Like a new config section for default cli settings. What do you think?

Reply:
it would indeed be super useful :+1: